device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/PnpHelpers.java

```diff
@@ -15,10 +15,11 @@
 
 import static java.util.Collections.singleton;
 import static java.util.Collections.singletonMap;
+
 /*
  A helper class for formatting command requests and properties as per plug and play convention.
 */
-public class PnpHelper {
+public class PnpHelpers {
```

Comment:
Let's call it PnpConvention since you have it in the samples.* package.

Reply:
I have addressed it. Thanks